projects/canopy/src/lib/details/details.notes.ts

```diff
@@ -3,8 +3,11 @@ export const notes = `
 
 
 ## Purpose
-The Details component reduces page clutter and cognitive load by letting users reveal more information as and when they need it.
+The Details component reduces page clutter and cognitive load by letting users reveal more information as and when they need it. It can also be used to communicate import imformation in a collapsed layout, similar to an Alert.
```

Comment:
```suggestion
The Details component reduces page clutter and cognitive load by letting users reveal more information as and when they need it. It can also be used to communicate important information in a collapsed layout, similar to an Alert.
```

Reply:
Done ✅